FWCore/Catalog/interface/FileLocator.h

```diff
@@ -1,22 +1,31 @@
 #ifndef FWCore_Catalog_FileLocator_h
 #define FWCore_Catalog_FileLocator_h
 
+#include "FWCore/Catalog/interface/SiteLocalConfig.h"
 #include <string>
 #include <list>
 #include <map>
 #include <utility>
 #include <regex>
 #include "tinyxml2.h"
+#include <boost/property_tree/ptree.hpp>
 
 namespace edm {
 
   class FileLocator {
   public:
+    explicit FileLocator(
+        edm::CatalogAttributes const& catAttr,
+        unsigned iCatalog = 0,
+        std::string storageDescriptionPath = std::
+            string());  //storageDescriptionPath is used to override path provided by SiteLocalConfig. This is used in FileLocator_t.cpp tests
```

Comment:
Please move the comment to the preceding line for better formatting. I'd also suggest to use
```cpp
std::string const& storageDescriptionPath = std::string()
// or
std::string const& storageDescriptionPath = ""

```